### DISCUSSION TITLE
there is no mort.exe

### DISCUSSION THREAD

**Discussion by Mottosann** [killkimno/MORT#33] [Q&A] [answered]
there is no mort.exe, did anyone managed to use mort properly, how?

**Comment by kmonkeyhead** [comment, marked as answer]
here is last version download link
https://github.com/killkimno/MORT/releases/download/1.276/MORT.1.276v.-.20240228.zip

If still dosent exist exe file, please check your antivirus.
false positive may cause  automatic remove exe file
